CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+# 2.6.2
+
+## User interface
+
+ * The `CEPH_KEYRING` environment variable no longer needs to be manually
+   specified; it can be passed in as the `--ceph-keyring` command-line
```

Comment:
Perhaps not. I had it like that originally, can't remember why I changed it. Will update.